### PR TITLE
[1.13.x] [RHPAM-4302] : RHPAM Kogito Operator image does not start

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Jira issue : https://issues.redhat.com/browse/RHPAM-4302

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

File: bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ metadata:
           "apiVersion": "rhpam.kiegroup.org/v1",
           "kind": "KogitoBuild",
           "metadata": {
-            "name": "process-quarkus-example"
+            "name": "ruleunit-quarkus-example"
```
Comment (davidesalerno) [inline bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:11]: Is this replace intentional?

Comment (vaibhavjainwiz) [reply bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:11]: I am suprised why these changes are missing in first place itself. Examples are already renamed to `ruleunit`.https://github.com/kiegroup/kogito-operator/blob/e93ee6989c548a32e26959fae6d22f7acc75e923/config/samples/rhpam/rhpam_v1_kogitobuild.yaml#L4

Comment (vaibhavjainwiz) [reply bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml:11]: For now I just run `make vet` cmd and it generate me a new CSV file.

**Comment by radtriste** [comment]
@vaibhavjainwiz I guess we should also update the examples/rhpam folder with ruleunit. See https://github.com/kiegroup/rhpam-kogito-operator/tree/1.11.x/example

**Comment by radtriste** [comment]
or I can create another issue for that (non blocking)

**Comment by radtriste** [comment]
ERRATUM: examples are fine ... I don't know what I was looking ...

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
@davidesalerno can you check why the `/jenkins test` hook to kogito operator check did not work ?

**Comment by radtriste** [comment]
@vaibhavjainwiz please merge https://github.com/vaibhavjainwiz/kogito-operator/pull/1 and restart the RHPAM jenkins test
This should fix a problem in the creation of image stream in BDD tests

**Comment by radtriste** [comment]
checking why the operator could not be installed ...

**Comment by radtriste** [comment]
RHPAM CI tests working correctly :)

@davidesalerno are we good to merge ?

**Pull request merged**